creator/ingest_runs/manifests.py

```diff
@@ -1,334 +0,0 @@
-
-    df = pd.concat(dfs)
-    df["Filename"] = df["Key"].map(lambda x: os.path.split(x)[-1])
-    return df
-
-
-def load_harmonized_genomic_files(genomic_df, study_id):
-    """
-    Load data for the harmonized genomic files found in the manifest into the
-    DataService (this data is in _genomic_df_).
-    This step must occur before the others so that we can use the
-    kf_ids created here for the rest of the process. After ingestion, grab the
-    generated kf_ids from the ingest lib cache, join them to _genomic_df_.
-    Then use _genomic_df_ to create genomic-file biospecimen links and load
-    them as well. Return _genomic_df_.
-    """
-    genomic_df = munge_genomic_df(genomic_df)
-    genomic_cache = load_entities(
-        GEN_FILE,
-        genomic_df.drop(CONCEPT.BIOSPECIMEN.ID, axis=1),
-        study_id,
-    )
-    genomic_df = get_genomic_file_kf_ids(genomic_cache, genomic_df)
-    
-    biospec_gen_df = genomic_df[[
-        CONCEPT.GENOMIC_FILE.ID,
-        CONCEPT.BIOSPECIMEN.ID,
-        CONCEPT.GENOMIC_FILE.TARGET_SERVICE_ID,
-    ]]
-    biospec_gen_df[CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID] = biospec_gen_df[
-        CONCEPT.BIOSPECIMEN.ID
-    ]
-    biospec_gen_df[CONCEPT.BIOSPECIMEN_GENOMIC_FILE.VISIBLE] = True
-
-    # Create genomic-file biospecimen links
-    _ = load_entities(BIO_GEN_FILE, biospec_gen_df, study_id)
-    return genomic_df
-
-
-def get_genomic_file_kf_ids(genomic_cache, genomic_df):
-    """
-    Takes genomic_df and appends the genomic-file kf-ids which are created
-    after they're ingested. The ingest library stores the kf-ids in a cache
-    which we must query. Returns genomic_df with a new kf-id column.
-    """
-    cache_dict = {
-        ast.literal_eval(key)['external_id']: value
-        for key, value in genomic_cache[GEN_FILE].items()
-    }
-    hash_df = pd.DataFrame(
-        list(cache_dict.items()),
-        columns=[
-            CONCEPT.GENOMIC_FILE.ID, CONCEPT.GENOMIC_FILE.TARGET_SERVICE_ID
-        ],
-    )
-    genomic_df = genomic_df.merge(
-        right=hash_df,
-        on=CONCEPT.GENOMIC_FILE.ID,
-        how='inner',
-    )
-    return genomic_df
-
-
-def load_entities(entity_type, df, study_id):
-    """
-    Load data of type _entity_type_ into the DataService from _df_.
-    """
-    target_service_base_url = LOCALHOST
-    app_settings = settings.load()
-    loader = LoadStage(
-        app_settings.TARGET_API_CONFIG,
-        target_service_base_url,
-        [entity_type],
-        study_id,
-        cache_dir=os.getcwd(),
-    )
-    loader.run({entity_type: df})
-
-    return loader.uid_cache
-
-
-def munge_genomic_df(df):
-    """
-    Clean up the genomic-file DataFrame and get it ready to be ingested by
-    removing unneeded columns and standardizing the column names.
-    """
-    df['Hashes'] = df.apply(lambda row: {"ETag": row['ETag']}, axis=1)
-    df['urls'] = df.apply(lambda row: [row['Filepath']], axis=1)
-    df['file_type'] = df.apply(
```

Comment:
This would have been hard for you to know without doing a bunch of ingests, but getting the file extension for the genomic files is a little tricky. There is a function in the s3_scrape_config.py which does this, so I replaced your lambda with that function. See _prep_harmonized_gf_df in creator/ingest_runs/genomic_data_loader.py